kahuna/public/js/search/structured-query/query-suggestions.js

```diff
@@ -79,6 +79,13 @@ const isSearch = [
 
 querySuggestions.factory('querySuggestions', ['mediaApi', 'editsApi', function(mediaApi, editsApi) {
 
+    window._clientConfig.fieldAliases.
```

Comment:
I think this would be better moved to the definition of filterFields, it's a little uncomfortable that it's exported and then mutated in the factory. Would also probably go for a filter/map combo.

Reply:
We had a look at this this morning and think that this is a better way of expressing it:
```javascript
const fieldAliases = window._clientConfig.fieldAliases
  .filter(entry => entry.displaySearchHint)
  .map(entry => entry.alias);

// FIXME: get fields and subjects from API
export const filterFields = [
    'by',
    'category',
    'city',
    'copyright',
    'country',
    'credit',
    'description',
    'fileType',
    'illustrator',
    'in',
    'keyword',
    'label',
    'location',
    'person',
    'source',
    'state',
    'subject',
    'supplier',
    'uploader',
    'usages@<added',
    'usages@>added',
    'usages@platform',
    'usages@status',
    'usages@reference',
    'has',
    'croppedBy',
    'filename',
    'photoshoot',
    'leasedBy',
    'is',
    ... fieldAliases
].sort();
```

Reply:
Thanks Simon. I will apply this right away.